finetuner-workflow/finetune-workflow.yaml

```diff
@@ -109,34 +151,50 @@ spec:
     # Training GPU - A40, 48gb VRAM
     - name: trainer_gpu
       value: 'A40'
+    # Training number of GPUs
+    - name: trainer_gpus
+      value: 1
+    # Training number of CPU cores
+    - name: trainer_cores
+      value: 8
+    # Training compute RAM in gigabytes
+    - name: trainer_ram
+      value: 192
     # Inference GPU - RTX A5000, 24gb VRAM
     - name: inference_gpu
       value: 'RTX_A5000'
     # Container images -- generally, don't alter this.
     - name: model_downloader_image
       value: 'ghcr.io/wbrown/gpt_bpe/model_downloader'
     - name: model_downloader_tag
-      value: 'e2ef65f'
+      value: 'fd6ec51'
+    - name: tokenizer_image
+      value: 'ghcr.io/wbrown/gpt_bpe/dataset_tokenizer'
+    - name: tokenizer_tag
+      value: 'fd6ec51'
```

Comment:
This image should be updated to the newest when [this PR](https://github.com/wbrown/gpt_bpe/pull/34) gets merged.